rsql-jpa/src/main/java/io/github/perplexhub/rsql/RSQLJPASupport.java

```diff
@@ -40,6 +43,29 @@ public RSQLJPASupport(Map<String, EntityManager> entityManagerMap, Map<EntityMan
 		RSQLVisitorBase.setEntityManagerDatabase(entityManagerDatabase);
 	}
 
+	/**
+	 * Set the escape character for LIKE operation
+	 * @param likeEscapeChar escape character
+	 */
+	public static void setLikeEscapeChar(char likeEscapeChar) {
+		RSQLJPASupport.LIKE_ESCAPE_CHAR = likeEscapeChar;
+	}
+
+	/**
+	 * Get the escape character for LIKE operation
+	 * @return escape character
+	 */
+	public static char getLikeEscapeChar() {
+		return RSQLJPASupport.LIKE_ESCAPE_CHAR;
+	}
+
+	/**
+	 * Reset the escape character for LIKE operation to default, which is '$'
+	 */
+	public static void setDefaultLikeEscapeChar() {
+		RSQLJPASupport.LIKE_ESCAPE_CHAR = DEFAULT_LIKE_ESCAPE_CHAR;
+	}
+
```

Comment:
It's a backward incompatible change. We are actually using it in production. I would suggest that default behavior stays the same, but nevertheless it's a useful configuration to have.

Frankly, I don't quite like the idea of having configuration in static fields in such manner. Users can run into data race here. This is a global mutable variables which is not considered best practice in general and especially in Spring environment. In this case I would suggest to transfer this configuration to `io.github.perplexhub.rsql.QuerySupport`.

Reply:
I've missed that point, it's now fixed according your recommandations.